.github/workflows/config/node-release.yaml

```diff
@@ -1,11 +1,11 @@
 release:
   branching:
     execution:
-      time: "22:00:00"
+      time: "23:00:00"
```

Comment:
We should not chnage this for final tags afaik. This is only for creating release branches